adding graphmap section (#154)
kg-construct/rml-core: spec/docs/graphmap.md

# Assigning Triples to Named Graphs

Each triple generated from an [=RML mapping=]() is placed into one or more graphs of the [=output dataset=](). Possible target graphs are the unnamed [=default graph=](), the [=IRI=]()-named [=named graphs=](), and [=blank node=]() named [=named graphs=]()

> Any [=subject map=]() or [=predicate-object=]() map MAY have one or more associated graph maps. They are specified in one of two ways:

Reviewer: ```suggestion
Any [=subject map=]() or [=predicate-object=]() map MUST have zero or more associated graph maps. They are specified in one of two ways:
```

1. using the rml:graphMap property, whose value MUST be a [=graph map=](),
2. using the [=constant shortcut property=]() rml:graph.

Graph maps are themselves [=term maps=](). When [=RDF triples are generated=](), the set of target graphs is determined by taking into account any graph maps associated with the subject map or predicate-object map.

If a [=graph map=]() generates the special IRI rml:defaultGraph, then the target graph is the [=default graph=]() of the [=output dataset=]().

In the following [=subject map=]() example, all generated RDF triples will be stored in the named graph ex:DepartmentGraph.

```turtle
[]  rml:subjectMap [
      rml:template "http://data.example.com/department/{DEPTNO}";
      rml:graphMap [ rml:constant ex:DepartmentGraph ];
].

> ```

Reviewer: Examples should be formatted like other examples in spec. Using `<aside ...>`

Author: done

# Assigning Triples to Named Graphs

> Each triple generated from an [=RML mapping=]() is placed into one or more graphs of the [=output dataset=](). Possible target graphs are the unnamed [=default graph=](), the [=IRI=]()-named [=named graphs=](), and [=blank node=]() named [=named graphs=]()

Reviewer: ```suggestion
Each triple generated from an [=RML mapping=] is placed into one or more graphs of the [=output dataset=]. Possible target graphs are the unnamed [=default graph=], the [=IRI=]-named [=named graphs=], and [=blank node=] named [=named graphs=]
```

> Any [=subject map=]() or [=predicate-object=]() map MUST have zero or more associated graph maps. They are specified in one of two ways:

Reviewer: ```suggestion
Any [=subject map=] or [=predicate-object=] map MUST have zero or more associated <dfn>graph maps</dfn>. They are specified in one of two ways:
```

> 1. using the rml:graphMap property, whose value MUST be a [=graph map=](),

Reviewer: ```suggestion
1. using the `rml:graphMap` property, whose value MUST be a [=graph map=],
```

> 2. using the [=constant shortcut property=]() rml:graph.

Reviewer: ```suggestion
2. using the [=constant shortcut property=] `rml:graph`.
```

> Graph maps are themselves [=term maps=](). When [=RDF triples are generated=](), the set of target graphs is determined by taking into account any graph maps associated with the subject map or predicate-object map.

Reviewer: ```suggestion
[=Graph maps=] are themselves [=term maps=]. When [=RDF triples are generated=], the set of target graphs is determined by taking into account any [=graph maps=] associated with the [=subject map=] or [=predicate-object map=].
```

> If a [=graph map=]() generates the special IRI rml:defaultGraph, then the target graph is the [=default graph=]() of the [=output dataset=]().

Reviewer: ```suggestion
If a [=graph map=] generates the special IRI `rml:defaultGraph`, then the target graph is the [=default graph=] of the [=output dataset=].
```
